resources/kubernetes/operator/helm/values.yaml

```diff
@@ -1,24 +1,28 @@
+# For advanced configuration options, refer to the [official OpenTelemetry Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml)
```

Comment:
Are markdown style links common in Yaml files?

Reply:
Not sure, but it seems they are really handy when rendered with VS Code. @mlunadia might have more context